config/initializers/ios_apn.rb

```diff
@@ -0,0 +1,25 @@
+if Rails.env.production?
+  IOS_APN = ENV["IOS_APN"] || Houston::Client.production
+
+  certificate = "config/ios_production_certificate.pem"
+  if File.exist? certificate
+    IOS_APN.certificate = File.read(certificate)
+  end
+
+  passphrase = "config/ios_production_passphrase.txt"
+  if File.exist? passphrase
+    IOS_APN.passphrase = File.read(passphrase)
+  end
```

Comment:
There is no persistent filesystem, so you would need to manually upload them on each deploy. I suggest storing the contents in the environment instead.